src/Appwrite/Template/Template.php

```diff
@@ -5,8 +5,57 @@
 use Exception;
 use Utopia\View;
 
+use function PHPUnit\Framework\isReadable;
+
 class Template extends View
 {
+
+    /**
+     * fromFile
+     *
+     * Creates a new Template() from the file at $path
+     * 
+     * @param string $path
+     *
+     * @return self
+     * 
+     */
+    public static function fromFile(string $path): self
+    {
+        if (!\is_readable($path)) {
+            throw new Exception("$path view template is not readable.");
+        }
+
+        $template = new Template();
+        return $template->setPath($path);
+    }
+
+
+    /**
+     * @var string
+     */
+    protected string $html = ''; 
+
+    /**
+     * fromHtmlString
+     *
+     * Creates a new Template() using raw HTML.
+     * 
+     * @param string $html 
+     *
+     * @return self
+     * 
+     */
+    public static function fromHtmlString(string $html): self
```

Comment:
```suggestion
    public static function fromString(string $html): self
```

HTML should be completely optional.

Reply:
Done :+1: